ci-operator/config/openshift-kni/mixed-cpu-node-plugin/openshift-kni-mixed-cpu-node-plugin-main.yaml

```diff
@@ -46,7 +46,11 @@ tests:
     test:
     - as: e2e-test
       cli: latest
-      commands: E2E_SHARED_CPUS=0 E2E_SETUP=TRUE E2E_TEARDOWN=TRUE make test-e2e
+      commands: E2E_IMAGE_NAME=$IMAGE_NAME E2E_SHARED_CPUS=0 E2E_SETUP=TRUE E2E_TEARDOWN=TRUE
+        make test-e2e
+      dependencies:
+      - env: IMAGE_NAME
+        name: mixed-cpu-node-plugin
```

Comment:
this is the incantation which make the CI use the image just built, right?

Reply:
The reason for the failure is because the test code is not updated to use this image yet.

Reply:
nice, thanks for the ref